### DISCUSSION TITLE
Import / Export actions DatabaseNotification is required

### DISCUSSION THREAD

**Discussion by ytorbyk** [filamentphp/filament#13876] [Ideas]
Is there a reason the Database Notification flag isEventDispatched is set to true?
Can it be configurable to false?
https://github.com/filamentphp/actions/blob/3.x/src/Concerns/CanImportRecords.php#L310

What if the application we do not want to use Broadcast Notifications but still want to use import/export?

It would be a nice small improvement.

**Comment by ronssij** [comment]
@ytorbyk AFAIK for Import , I beleive it is by default because If the import fails it will allow you to download in the notification 
panel/component with a new csv files that contains the failed row(s) and the reason why it failed.

![image](https://github.com/user-attachments/assets/581e4ce4-28b2-48bc-9988-d7f79684139d)

**Reply by ronssij** [reply]
> This feature uses [job batches](https://laravel.com/docs/queues#job-batching) and [database notifications](https://filamentphp.com/docs/3.x/notifications/database-notifications#overview), so you need to publish those migrations from Laravel. Also, you need to publish the migrations for tables that Filament uses to store information about imports:

Since its required and you don't want to include the database notifcation, you can still [customize the list](https://filamentphp.com/docs/3.x/notifications/customizing-notifications) to exclude it on the list.

or you can make an model observer to delete the record on notification to after it was created

**Comment by ytorbyk** [comment]
@ronssij The notification will be sent even without "isEventDispatched: true". As I understand the difference will be only "will the notification be sent real time or after page is refreshed".
